source/Track.cpp

```diff
@@ -0,0 +1,97 @@
+/* Track.cpp
+Copyright (c) 2022 by RisingLeaf
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "Track.h"
+
+#include "DataNode.h"
+
+#include <algorithm>
+#include <iostream>
+
+using namespace std;
+
+
+
+Track::Track(const DataNode &node)
+{
+	Load(node);
+}
+
+
+
+void Track::Load(const DataNode &node)
+{
+	if(!name.empty())
+	{
+		node.PrintTrace("Error: Duplicate definition of track:");
+		return;
+	}
+	name = node.Token(1);
+
+	for(const DataNode &child : node)
+	{
+		const string &key = child.Token(0);
+		bool hasValue = child.Size() >= 2;
+		if(key == "volume" && hasValue)
+			volumeModifier = clamp<double>(child.Value(1), -1., 1.);
+		else if(child.Token(0) == "idle" && child.Size() >= 2)
+			idleTitle = child.Token(1);
+		else if(child.Token(0) == "combat" && child.Size() >= 2)
+			combatTitle = child.Token(1);
+		else if(child.Token(0) == "landed" && child.Size() >= 2)
```

Comment:
The PR documentation has this listed as `landing`, while the original design doc has it as `landed` along with the code here. Should make it consistent one way or the other.

Reply:
Should probably match mission syntax

Reply:
landing might convey the idea that it only happens once when you first land and then not anymore (which would be fine ig but isnt whats happening?)

Reply:
I updated the PR description to `landed`.  If it changes in code I guess let me know and I can update it again.

Reply:
Resolving since I think this was addressed by updating PR description.